pom.xml

```diff
@@ -672,6 +679,12 @@
             <artifactId>jaxb-api</artifactId>
             <version>${jaxb-api.version}</version>
          </dependency>
+         <dependency>
+            <groupId>io.netty</groupId>
+            <artifactId>netty-all</artifactId>
```

Comment:
This dependency is not required because it is already part of `<artifactId>netty-bom</artifactId>` from the `spring-boot-dependencies`